spec/services/activitypub/fetch_remote_status_service_spec.rb

```diff
@@ -125,7 +125,34 @@
 
         expect(status).to_not be_nil
         expect(status.url).to eq 'https://foo.bar/@foo/1234'
-        expect(strip_tags(status.text)).to eq "Let's change the worldhttps://foo.bar/@foo/1234"
+        expect(strip_tags(status.text)).to eq "Let's change the world\n\nhttps://foo.bar/@foo/1234"
+      end
+    end
+
+    context 'with Event object that contains a summary' do
+      let(:object) do
+        {
+          '@context': 'https://www.w3.org/ns/activitystreams',
+          id: 'https://foo.bar/@foo/1234',
+          type: 'Event',
+          name: "Let's change the world",
+          startTime: '2024-01-31T20:00:00.000+01:00',
+          location: {
+            type: 'Place',
+            name: 'FooBar – The not converted location',
+          },
+          content: 'The not converted description of the event object.',
+          summary: 'We meet on January 31st at 8pm in the FooBaar!',
+          attributedTo: ActivityPub::TagManager.instance.uri_for(sender),
+        }
+      end
+
+      it 'creates status' do
+        status = sender.statuses.first
+
+        expect(status).to_not be_nil
+        expect(status.url).to eq 'https://foo.bar/@foo/1234'
+        expect(strip_tags(status.text)).to eq "Let's change the world\n\nWe meet on January 31st at 8pm in the FooBaar!\n\nhttps://foo.bar/@foo/1234"
```

Comment:
Both this added coverage and the pre-existing are all using strip_tags in their assertions ... which I think is mostly fine for the purposes we had. That said - since this code is adding the `<h2>`-wrapped value ... should we assert for its presence (and/or any other ways it may be converted elsewhere?)

Reply:
@Menrath As far as I can see there is still no test that the thing you set out to to, preserve HTML in summaries, now works.

Would you like to add one? Or would you prefer one of us doing it?